### PR TITLE
Add Item id for Desert Treasure diamonds

### PR DESCRIPTION
**Pull request by DavidLuong98**
See issue #14762 

Adding labels for the 4 diamonds.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java [hunk 1]
```diff
@@ -254,6 +254,10 @@
 	DRAGONSTONE(Type.GEM, "Dragon", "DR", ItemID.UNCUT_DRAGONSTONE, ItemID.DRAGONSTONE),
 	ONYX(Type.GEM, "Onyx", "ON", ItemID.UNCUT_ONYX, ItemID.ONYX),
 	ZENYTE(Type.GEM, "Zenyte", "Z", ItemID.UNCUT_ZENYTE, ItemID.ZENYTE),
+	SHADOW_DIAMOND(Type.GEM, "Shadow Diamond", "SHD", ItemID.SHADOW_DIAMOND),
+	BLOOD_DIAMOND(Type.GEM, "Blood Diamond", "BD", ItemID.BLOOD_DIAMOND),
+	ICE_DIAMOND(Type.GEM, "Ice Diamond", "SD", ItemID.ICE_DIAMOND),
```
Comment (Skretzo) [inline runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java:259]: Why is the short name `"SD"` for the ice diamond?

Comment (DavidLuong98) [reply runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java]: I had a typo. thanks for catching that. I pushed a fix

**Comment by Adam-** [comment]
These long names you've chosen are probably too long and would overlap the adjacent items. See eg https://github.com/runelite/runelite/pull/14735#issuecomment-1062540681. Test it out.

**Comment by Nightfirecat** [comment]
Why not mark them with the type of diamond they are? I feel like Shadow/Blood/Ice/Smoke would be much more readable and understandable than ShDia/BlDia/IceDia/SmDia

**Comment by DavidLuong98** [comment]
> Why not mark them with the type of diamond they are? I feel like Shadow/Blood/Ice/Smoke would be much more readable and understandable than ShDia/BlDia/IceDia/SmDia

@Nightfirecat I'll change it. At first I thought that since the person is colorblind, it would be useful to explicitly mention that these are diamonds, since other gems have the same shape. Though, no other game contents are Shadow/Blood/Ice/Smoke gems, so what you are saying makes sense.

**Pull request merged**